apps/alpha-test-app/src/components/layout-demo/MultiSettingsGroup.vue

```diff
@@ -0,0 +1,45 @@
+<script lang="ts" setup>
+import { OnyxCheckboxGroup, type SelectionOption } from "sit-onyx";
+import { computed } from "vue";
+import type { Settings } from "./LayoutSettings.vue";
+
+const activeSettings = defineModel<Settings>();
+
+const props = defineProps<{
+  headline: string;
+  options: SelectionOption<undefined>[];
+  horizontal?: boolean;
+}>();
+
+const options = computed<SelectionOption<Settings>[]>(() =>
+  props.options.map(
+    (option) => ({ ...option, value: { [option.id]: true } }) as SelectionOption<Settings>,
```

Comment:
Why is the value added here? Is it used somewhere? The option id is used for the modeValue, `value` is only to attach custom data to the option.

Also, the checkbox currently does not support the value at all until #376

Reply:
Same for the SingleSettingsGroup.vue

Reply:
I previously used value instead of the ID. will check

Reply:
rewrote it so it only happens in the setter now in SingleSettingsGroup. Removed it from MultiSettingsGroup